courses/free-courses-pt_BR.md

```diff
@@ -72,7 +72,8 @@
 ### CSS
 
 * [Introdução à linguagem CSS](https://www.udemy.com/introducao-a-linguagem-css/) - Diego Mariano (Udemy)
-* [Novo curso HTML5 e CSS3: 100% atual](https://www.youtube.com/playlist?list=PLHz_AreHm4dkZ9-atkcmcBaMZdmLHft8n) - Curso em Vídeo (Youtube)
+* [Novo curso HTML5 e CSS3: Módulo 1 - 100% atual](https://www.youtube.com/playlist?list=PLHz_AreHm4dkZ9-atkcmcBaMZdmLHft8n) - Curso em Vídeo (Youtube)
```

Comment:
typo in titles?

Reply:
Hi,
I'm sorry but I don't understand your question. But I try to explain what I did.
I updated the name of the course because it has more than 1 part and each part is in a different place/link. 
If I made a mistake, I apologize

Reply:
Improves #4445

An approach adding instructor and fixing trademarks

```suggestion
* [Curso completo e atual de HTML5 e CSS3 - Módulo 1 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dkZ9-atkcmcBaMZdmLHft8n) - Professor Gustavo Guanabara, Curso em Vídeo (YouTube)
```

Reply:
Some questions...

- Do exist the playlist to the 3,4,5 modules?
- Why duplicate in HTML category? Maybe it's the time to join both like in other files

Reply:
Hi,
I understand your suggestion for a change. I can follow your suggestion and change whatever it takes.
About your questions:
1 - Yes, there are more modules. However, only modules 1 and 2 are complete on Youtube. Module 3 is ongoing, with 3 lessons per week. Modules 4 and 5 are still in production.
2 - I duplicated in the HTML category because the previous version was duplicated. As I updated in CSS category, I updated in HTML category too.
As I'm new on GitHub and started coding 3 months ago, I just tried to use the same template that was in the code, just updating.

Reply:
If exist url for any of 3,4,5... feel free to add them. If not... added to backlog as well :hugs:

If is easier for you, duplicate it. In anyway, A repo reorganization will be performed when hacktoberfest ends.

Apply suggested patch and mark this conversation as resolved. Request for a review after.